mpas_analysis/config.default

```diff
@@ -1694,7 +1697,7 @@ normArgsDifference = {'vmin': -0.2, 'vmax': 0.2}
 ## results and WOA climatological data
 
 # comparison grid(s) ('latlon', 'antarctic', 'arctic') on which to plot analysis
-comparisonGrids = ['arctic', 'antarctic']
+comparisonGrids = ['latlon']
```

Comment:
Rather than disabling WOA, I put it on the lat/lon grid by default.  Somewhat redundant with Argo, but I think that's okay.